code/game/objects/items/devices/enginepicker.dm

```diff
@@ -0,0 +1,80 @@
+/*
+//////////
+	Item meant to spawn one of the three (Tesla / Singularity / Supermatter) engines on-station at round-start.
+	Should be found in the CE's office. Not access-restricted.
+//////////
+*/
+
+/obj/item/enginepicker
+	name = "Bluespace Engine Delivery Device"
+	desc = "With all of the innovations in power-generating science, Nanotrasen has decided to invent a per-station bluespace-based delivery system for a unique engine of the Chief Engineer's choice. Only one choice can be made. Device self-destructs on use."
```

Comment:
This is really damn bloated

Reply:
Shortened the description to be more concise.